### DISCUSSION TITLE
Notifications add-on

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#138] [Ideas]
- https://github.com/WordPress/wp-notify/tree/develop
- https://make.wordpress.org/core/2019/08/05/feature-project-proposal-wp-notify/
- https://wordpress.org/plugins/notification/

_Related GitHub issue:_ https://github.com/pronamic/wp-pronamic-pay/issues/7
_Internal Basecamp to-do:_ https://basecamp.com/1810084/projects/18117394/todos/442319992

**Comment by remcotolsma** [comment]
- https://github.com/pronamic/pronamic-pay-notification-add-on/issues/1

**Comment by rvdsteege** [comment]
ℹ️ Support for https://wordpress.org/plugins/notification/ is available in Pronamic Pay Premium.
